framework/tlvf/tlvf.py

```diff
@@ -583,6 +583,62 @@ def addStructParam(self, obj_meta, param_name, param_type, param_type_info, para
         else:
             self.insertLineH(obj_meta.name, self.CODE_STRUCT_INSERT, line)
 
+    ##########################################################################################
+    # Empty class member initialization support
+    # 
+    # In order to make sure that a class that contains a class member is fully
+    # initialized, the virtual method isPostInitSucceeded is overridden to
+    # validate that and return true or false to indicate.
+    #
+    # For each TLV that contains a member of a type class, a private bool member
+    # is created (for each class member) which indicates if that member class is
+    # initialized. This bool variable is initialized to FALSE.
+    #
+    # When the user calls the add_member method, the value of the bool is set to TRUE.
+    # In those cases, the method isPostInitSucceeded is overridden to include
+    # more conditions to validate that each of the members of the type class
+    # in TLV is fully initialized.
+    #
+    ##########################################################################################
+    def overrideIsPostInitSucceeded(self, obj_meta, param_name,lines_h):
+        #for each TLV member of type class add a bool variable to indicate if it is initialized 
+        lines_h.append("bool m_%s_init = false;" % (param_name))
+        lines_cpp = []
+        #if this is the first member of type class override the method 
+        if not obj_meta.marker_post_init_method_added:
+            #Add decleration in header file
+            line_h = "bool isPostInitSucceeded();" 
+            self.insertLineH(obj_meta.name, self.CODE_CLASS_PUBLIC_FUNC_INSERT, line_h)
+                    
+            #Override isPostInitSucceeded method to verify the class is completly initialized
+            lines_cpp.append("bool %s::isPostInitSucceeded() {" % (obj_meta.name))
+            #if this member is not added by the user return false
+            lines_cpp.append("%sif(!m_%s_init) {" % (self.getIndentation(1) ,param_name))
```

Comment:
we don't run clang-format here but we can at least try to align to our coding rules, so add a space after the `if`:
```
lines_cpp.append("%sif (!m_%s_init) {" % (self.getIndentation(1) ,param_name))
```

Same comment for other places you introduced an `if` without a space.